envi/qt/memcanvas.py

```diff
@@ -324,6 +324,8 @@ def getNavTargetNames():
 
 
 def initMemSendtoMenu(expr, menu):
-    for name in set(getNavTargetNames()):
+    navtgts = getNavTargetNames()
+    navtgts.sort()
```

Comment:
So this removes the deduplication the sort gives. Is that intended and is the `set` still pertinent?

Reply:
your question is totally valid.
no, duplication shouldn't be a problem.  `getNavTargetNames()` sends a request to all gui components and they respond only once.

btw, `getNavTargetNames()` is a neat exercise of gui event messaging... that i'd like to make more concrete sometime in the future.  it's unnecessarily race-y, and is better served by digging through instantiated nav windows.  but we can do that later, especially since it hasn't been a problem so far